style.css

```diff
@@ -0,0 +1,206 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-weight: bold;
```

Comment:
Doesn't this make all the contents in the page bold? Interesting start to style the page